packages/one-bx-shell-app/src/assets/config/plugins-bak.json

```diff
@@ -0,0 +1,164 @@
+[
```

Comment:
Ingore these comments if this file is something I should skip.

Reply:
Skip this file